.env.example

```
@@ -0,0 +1,3 @@
+KNUU_TIMEOUT=60m # The timeout for the tests
+KNUU_BUILDER=docker # The builder to use for building images. docker or kubernetes
+DEBUG_LEVEL=info # The debug level. debug, info, warn, error
```

I think this is called `LOG_LEVEL`

```suggestion
LOG_LEVEL=info # The debug level. debug, info, warn, error
```